src/devices/tuya.ts

```diff
@@ -5529,7 +5669,7 @@ const definitions: DefinitionWithExtend[] = [
         vendor: 'SUTON',
         description: 'Zigbee DIN RCBO energy meter',
         fromZigbee: [tuya.fz.datapoints],
-        toZigbee: [tuya.tz.datapoints],
+        toZigbee: [tzLocal.TZE204_threshold, tuya.tz.datapoints],
```

Comment:
Please use just `tuya.tz.datapoints` instead of `TZE204_threshold`

Reply:
OK, you mean to implement the logic in tuya.valueConvertor? Is there an example how to implement a setting logic for several exposed attributes behind a single dataPoint? Thanks

Reply:
Yes, here is an example to send data points directly from the converter (https://github.com/Koenkk/zigbee-herdsman-converters/blob/615eb00f66267429fa8434a233b39f2259af3d1a/src/lib/tuya.ts#L977). With this you ca also send multiple.